deploy/test-environments/delete_gcp_env.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ "$#" -lt 3 ]; then
+    echo "Missing params. Usage: $0 PROJECT_NAME PROJECT_NUMBER DEPLOYMENT1,DEPLOYMENT2,..."
+    exit 1
+fi
+
+DELETED_DEPLOYMENTS=()
+FAILED_DEPLOYMENTS=()
+PROJECT_NAME=$1
+PROJECT_NUMBER=$2
+shift 2
+GCP_DEPLOYMENTS=("$@")
+
+echo "Project Name: $PROJECT_NAME"
+echo "Project Number: $PROJECT_NUMBER"
+echo "GCP Deployments: ${GCP_DEPLOYMENTS[*]}"
+
+for DEPLOYMENT in "${GCP_DEPLOYMENTS[@]}"; do
```

Comment:
`${GCP_DEPLOYMENTS[@]}` changed from the original script (`$ALL_GCP_DEPLOYMENTS`) as per `shfmt` error